src/docs/self-hosted/external-storage.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: External Storage
+---
+
+In some cases, storing Sentry data on-disk is not really something people can do. Sometimes, it's better if they can offload it into some bucket storage (like AWS S3 or Google Cloud Storage).

```

Comment:
This seems a bit confusing, but I think adding another page later after this about "Unsupported Workflows" in which we can specify more about what kind of things that we can't offer support to (external Redis, external Postgres, installing third party plugins for extending some stuff).

See @azaslavsky's comment here https://github.com/getsentry/develop/pull/1269#discussion_r1598919946